src/containers/App.jsx

```diff
@@ -42,129 +42,106 @@ export const toFlagNames = (flagName, prefix = '') => {
       keys(flagName).map(key => toFlagNames(flagName[key], `${prefix}${key}.`))
     )
 }
-
-class App extends Component {
-  state = {
-    error: null,
-    status: IDLE
-  }
-
-  constructor(props, context) {
-    super(props, context)
-    this.store = context.store
-  }
-
-  componentDidMount() {
-    this.fetchContext()
-  }
-
-  async fetchContext() {
-    this.setState({
-      status: FETCHING_CONTEXT
-    })
-    const { client } = this.props
-    const context = await client.stackClient
-      .fetchJSON('GET', '/settings/context')
-      .catch(error => {
-        this.setState({
-          error,
-          status: IDLE
+const App = ({
+  // eslint-disable-next-line no-unused-vars
+  store,
+  client,
+  accounts,
+  konnectors,
+  triggers,
+  wallpaperFetchStatus,
+  wallpaperLink
+}) => {
+  const [status, setStatus] = useState(IDLE)
+  // eslint-disable-next-line no-unused-vars
+  const [error, setError] = useState(null)
+  const [contentWrapper, setContentWrapper] = useState(undefined)
+
+  useEffect(() => {
+    setStatus(FETCHING_CONTEXT)
+
+    async function fetchContext() {
+      return await client.stackClient
+        .fetchJSON('GET', '/settings/context')
```

Comment:
Since you're refactoring... We should not call stackClient.fetchJSON directly since it bypass CozyClient's redux store. 

```
client.query(Q('io.cozy.settings'). getById('context')
```